src/bsp/src/stm32/hal/src/usb.c

```diff
@@ -0,0 +1,41 @@
+#include "hal/usb.h"
+#include "usbd_cdc_if.h"
+
+uint8_t cbuffUsbData[CBUFF_USB_DATA_SIZE];
+
+uint8_t Usb_HasTxFinished(USBD_CDC_HandleTypeDef* hcdc){
+    bool ret = false;
+    while(hcdc->TxState != 0){}
+    if (hcdc->TxState == 0) {
+        ret = USBD_OK;
+    }else{
+        ret = USBD_FAIL;
+    }
+
+    return ret;
+}
+
+uint8_t Usb_Send_Data(const uint8_t* buf, uint16_t Len) {
```

Comment:
Change return type for the enum of `Usb_HasTxFinished()`

Reply:
changed to our own enum, USB_StatusTypeDef